gtsfm/loader/astronet_loader.py

```diff
@@ -0,0 +1,231 @@
+"""Simple loader class that reads a dataset with metadata formatted in the COLMAP style.
+
+Authors: Travis Driver
+"""
+
+import os
+from pathlib import Path
+from typing import Optional, Dict, Tuple, List
+
+from gtsam import Cal3Bundler, Pose3, Rot3, Point3, SfmTrack
+
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.loader_base import LoaderBase
+
+from thirdparty.colmap.scripts.python.read_write_model import read_model
```

Comment:
nit: typically we only import classes from modules, not individual function names. This is per the Google style guide: https://google.github.io/styleguide/pyguide.html#22-imports.

So I would prefer:
```python
import thirdparty.colmap.scripts.python.read_write_model as colmap_io
```
or
```python
import thirdparty.colmap.scripts.python.read_write_model as read_write_models
```

Reply:
+1

Reply:
Agreed. Fixed